### PR TITLE
Add Cashu wallet backend

### PR DESCRIPTION
**Pull request by callebtc**
Adds Cashu wallet backend through the REST API.

Depends on https://github.com/cashubtc/nutshell/pull/318

### CODE REVIEW

**Review by prusnak — CHANGES_REQUESTED** [review]
untested, code looks good

**Review by prusnak — APPROVED** [review]
untested, code looks good

File: lnbits/wallets/cashu.py [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        # run async_init avoiding the following error:
+        # asyncio.run() cannot be called from a running event loop
+
+    async def cleanup(self):
+        pass
+        # try:
+        #     await self.client.aclose()
+        # except RuntimeError as e:
+        #     logger.warning(f"Error closing wallet connection: {e}")
```
Comment (dni) [inline lnbits/wallets/cashu.py:45]: any reason for why not cleaning up the client?

**Comment by dni** [comment]
@callebtc the required PR seems to be merged, can you take a look again?

**Comment by motorina0** [comment]
@callebtc 
 - is this PR completed
 - is there a `cashu` docker image that we can use in our regtest

**Comment by arcbtc** [comment]
Bump @callebtc. Lets get this out of draft and ready to merge

**Comment by callebtc** [comment]
I won't have time to finish this PR unfortunately

**Pull request closed without merging**